src/components/Cart/CartListItem/CartListItem.scss

```diff
@@ -0,0 +1,136 @@
+@import '../../../styles/utils/_vars.scss';
+@import '../../../styles/utils/mixins';
+
+.cart-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
```suggestion

```
You have to use more complicated positioning here. Check the layout: -/+ buttons must be straight under the previous ones.